### PR TITLE
refactor(radar_tracks_msgs_converter): rework parameters

### PR DESCRIPTION
**Pull request by kaspermeck-arm**
## Description
Implement the ROS Node configuration layout described in https://github.com/orgs/autowarefoundation/discussions/3371 for the `radar_tracks_msgs_converter` package.

- Remove the default value from the source code in order to ensure all parameter values are passed from the parameter files.
- Create the schema

Original PR
* https://github.com/autowarefoundation/autoware.universe/pull/4911

## Tests performed
Package built and launch locally.
`colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release --packages-up-to radar_tracks_msgs_converter`

## Effects on system behavior
More reliable and faster parameter configuration file creation.

## Pre-review checklist for the PR author
The PR author **must** check the checkboxes below when creating the PR.
- [x] I've confirmed the [contribution guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/).
- [x] The PR follows the [pull request guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/).

## In-review checklist for the PR reviewers
The PR reviewers **must** check the checkboxes below before approval.
- [ ] The PR follows the [pull request guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/).
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author
The PR author **must** check the checkboxes below before merging.
- [x] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

File: perception/radar_tracks_msgs_converter/launch/radar_tracks_msgs_converter.launch.xml [hunk 1]
```diff
@@ -3,13 +3,13 @@
   <arg name="input/odometry" default="/localization/kinematic_state"/>
   <arg name="output/radar_detected_objects" default="output/radar_detected_objects"/>
   <arg name="output/radar_tracked_objects" default="output/radar_tracked_objects"/>
-  <arg name="param_path" default="$(find-pkg-share radar_tracks_msgs_converter)/config/radar_tracks_msgs_converter.param.yaml"/>
+  <arg name="config_file" default="$(find-pkg-share radar_tracks_msgs_converter)/config/radar_tracks_msgs_converter.param.yaml"/>
```
Comment (scepter914) [inline perception/radar_tracks_msgs_converter/launch/radar_tracks_msgs_converter.launch.xml:6]: [MUST]
This unnecessary interface changes affect many Autoware projects.
Please revert this change.

File: perception/radar_tracks_msgs_converter/launch/radar_tracks_msgs_converter.launch.xml [hunk 2]
```diff
@@ -3,13 +3,13 @@
...
 
   <node pkg="radar_tracks_msgs_converter" exec="radar_tracks_msgs_converter_node" name="radar_tracks_msgs_converter" output="screen">
     <remap from="~/input/radar_objects" to="$(var input/radar_objects)"/>
     <remap from="~/input/odometry" to="$(var input/odometry)"/>
     <remap from="~/output/radar_detected_objects" to="$(var output/radar_detected_objects)"/>
     <remap from="~/output/radar_tracked_objects" to="$(var output/radar_tracked_objects)"/>
-    <param from="$(var param_path)"/>
+    <param from="$(var config_file)"/>
```
Comment (scepter914) [inline perception/radar_tracks_msgs_converter/launch/radar_tracks_msgs_converter.launch.xml:13]: [MUST]
This unnecessary interface changes affect many Autoware projects.
Please revert this change.

**Review by scepter914 — CHANGES_REQUESTED** [review]

File: perception/radar_tracks_msgs_converter/launch/radar_tracks_msgs_converter.launch.xml (hunk 1 above)
Comment (mitsudome-r) [reply perception/radar_tracks_msgs_converter/launch/radar_tracks_msgs_converter.launch.xml:6]: @kaspermeck-arm 
Could you provide the reasons for this modification?

Like @scepter914 says, this would affect users who includes this radar_tracks_msgs_converter.launch.xml to their own project.
Since we don't want to disturb developers, let's keep this as and focus on node configuration, not launch configuration. If we need to unify the launch arguments as well, I think we should discuss this under https://github.com/autowarefoundation/autoware.universe/issues/3386 first.

Comment (kaspermeck-arm) [reply perception/radar_tracks_msgs_converter/launch/radar_tracks_msgs_converter.launch.xml:6]: @mitsudome-r 

To align with the original PR
* https://github.com/autowarefoundation/autoware.universe/pull/4911/files#diff-9328a941a658e3bda840cef66db3784d382a5e82e94560b6b31f6aef57091e81

and because in
* https://github.com/autowarefoundation/autoware.universe/commit/c53b81f3ad7314c53a27b3bc7721aaef6ec35c9d

it wasn't addressed.

I'll update the launch file and we can discuss naming convention when we start the ROS Launch dojo.

**Review by scepter914 — APPROVED** [review]
